app/views/media_objects/_item_view.html.erb

```diff
@@ -26,7 +26,9 @@ Unless required by applicable law or agreed to in writing, software distributed
 <% f_start, f_end = parse_media_fragment params['t'] %>
 <div class="row">
   <div class="col-sm-8" id="left_column">
-    <% if can_stream && @currentStream %>
+    <% if Settings.controlled_digital_lending.enable && !can_stream %>
+      <%= render 'embed_checkout' %>
+    <% elsif @currentStream %>
```

Comment:
I think this change was based on an earlier version of this file and needs to be updated to handle the latest in `develop`.

Reply:
So, I did base this change off what is current on [develop](https://github.com/avalonmediasystem/avalon/blob/fc7bba2bd8efd2436e872fa05dd260147dbbceba/app/views/media_objects/_item_view.html.erb#L29). I just flipped the conditional around because that was the easiest way I could conceptualize handling both cdl enabled and disabled without having to duplicate the whole block of render code.

Happy to change it if there's a better way.

Reply:
Sorry, I'll take a closer look.

Reply:
Ok, I'm seeing it now.  Nice.  I think this makes sense.